ci/playbooks/edpm/run.yml

```diff
@@ -15,11 +15,18 @@
         path: "{{ ansible_user_dir }}/ci-framework-data/artifacts/edpm-ansible.yml"
       register: edpm_file
 
+    - name: Save current zuul inventory
+      ansible.builtin.copy:
+        dest: "{{ ansible_user_dir }}/ci-framework-data/artifacts/edpm_zuul_inventory.yml"
+        src: "{{ inventory_file }}"
+
     - name: Run Podified EDPM deployment
       ansible.builtin.command:
         chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/ci-framework"
         cmd: >-
           ansible-playbook deploy-edpm.yml
+          -i {{ ansible_user_dir }}/ci-framework-data/artifacts/edpm_zuul_inventory.yml
```

Comment:
ceph playbook needs this inventory to deploy services in edpm/compute nodes.

Reply:
Yet another zuul inventory copy?

Reply:
Yeah, but I don't see an official one besides this https://github.com/openstack-k8s-operators/ci-framework/pull/642/files